1.setup.sh

```diff
@@ -65,7 +65,8 @@ idem "command -v curl" "sudo apt install curl"
 
 function init_aws () {
   curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
-  unzip awscliv2.zip
+  unzip ./awscliv2.zip
+  rm ./awscliv2.zip
```

Comment:
:pray: 

I also overlooked the unzip creating an `aws` folder which should probably also be cleaned up.  This begs the question whether this should be done in a temp folder...  You may want to clean up `~/aws` (as opposed to `~/.aws` which is used by the sdk) too.  Sorry for the oversight and thank you for helping improve this.